pkgs/development/python-modules/scikit-hep-testdata/default.nix

```
@@ -0,0 +1,53 @@
+{ lib
+, fetchFromGitHub
+, python
+, buildPythonPackage
+, importlib-resources
+, pyyaml
+, requests
+, pytestCheckHook
+, setuptools_scm
+, git
+}:
+
+buildPythonPackage rec {
+  pname = "scikit-hep-testdata";
+  version = "0.4.2";
+
+  # fetch from github as we want the data files
+  # https://github.com/scikit-hep/scikit-hep-testdata/issues/60
+  src = fetchFromGitHub {
+    owner = "scikit-hep";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-SxfYfOW/LWanY0zLKj7taYOnvMFxg2XJRWT1a8MdqoI=";
+
+    # for setuptools_scm
+    # https://github.com/scikit-hep/scikit-hep-testdata/issues/59
+    leaveDotGit = true;
+  };
+
+  nativeBuildInputs = [
+    git
+  ];
+  buildInputs = [
+    setuptools_scm
+  ];
+  propagatedBuildInputs = [
+    importlib-resources
```

For the latest Python releases we should use the `importlib.resources` module which is part of the Python standard library.

Replace `importlib-resources` with

```suggestion
     ++ lib.optionals (pythonOlder "3.9") [ importlib-resources ];
```